README.md

```diff
@@ -1,35 +1,35 @@
 # teamsnap-SDK-iOS
 TeamSnap iOS SDK
-For information on the TeamSnap API, and to get client tokens for your application check out http://developer.teamsnap.com
+For information on the TeamSnap API, and to get client tokens for your application check out [developer.teamsnap.com](http://developer.teamsnap.com).
 
 # Installing as a Framework
 Drag the framework into your frameworks group, and copy files. 
-make sure it appears under your Target->Build Phases->Link Binary With LIbraies  (happens automatically) AND
-target->General->Embedded Binaries (Does not happen automatically)
+make sure it appears under your `{target_name}->Build Phases->Link Binary With Libraries` (happens automatically) AND
+`{target_name}->General->Embedded Binaries` (Does not happen automatically)
 
-# Installation with CocoaPods
+# Setup
 
-[CocoaPods](http://cocoapods.org/) is a dependency manager for Objective-C, which automates and simplifies the process of using 3rd-party libraries in your projects. See the [Get Started](http://cocoapods.org/#get_started) section for more details.
```

Comment:
This section should be updated, not removed. 
Suggesting:
    pod 'TeamSnapSDK', :git => 'git@github.com:teamsnap/teamsnap-SDK-iOS', :branch => 'master' or a release specific branch to help avoid breaking changes.

Bonus points for getting this added to CocoaPods properly!